frontend/public/components/overview/index.tsx

```diff
@@ -275,6 +276,32 @@ const sortBuilds = (builds: K8sResourceKind[]): K8sResourceKind[] => {
   return builds.sort(byBuildNumber);
 };
 
+const overviewEmptyStateToProps = ({UI}) => ({activeNamespace: UI.get('activeNamespace')});
+
+const OverviewEmptyState = connect(overviewEmptyStateToProps)(({activeNamespace}) => (
+  <EmptyState>
+    <EmptyState.Title>
+      Get started with your project.
+    </EmptyState.Title>
+    <EmptyState.Info>
+      Add content to your project from the catalog of web frameworks, databases, and other components. You may also deploy an existing image or create or replace resources using YAML and JSON definitions.
+    </EmptyState.Info>
+    <EmptyState.Action>
+      <Link to="/catalog" className="btn btn-lg btn-primary">
+        Browse Catalog
+      </Link>
+    </EmptyState.Action>
+    <EmptyState.Action secondary>
+      <Link className="btn btn-default" to={`/deploy-image?preselected-ns=${activeNamespace}`} title="Deploy Image">
+        Deploy Image
+      </Link>
+      <Link className="btn btn-default" to={formatNamespacedRouteForResource('import', activeNamespace)} title="Perform an action">
+        Import YAML/JSON
```

Comment:
I think we should just say Import YAML. JSON is a YAML subest, so it's technically YAML. Import YAML would also be consistent with the add dropdown in the project bar.